lib/AST/ASTImporter.cpp

```diff
@@ -136,6 +136,15 @@ namespace clang {
       To->setIsUsed();
   }
 
+  void addDeclToDCIfVisible(Decl *From, Decl *To, DeclContext *ToDC,
```

Comment:
This is not the best name, because a Decl is visible in the DeclContext if the `LookupPtr` contains it. `containsDecl` is independent from that lookup table, it goes through the list of children declarations of the DC. See my comment below too.

Reply:
`AddDeclToDCIfContained` ?